sky/task.py

```diff
@@ -422,10 +430,39 @@ def from_yaml_config(
             task.set_outputs(outputs=outputs,
                              estimated_size_gigabytes=estimated_size_gigabytes)
 
-        resources = config.pop('resources', None)
-        resources = sky.Resources.from_yaml_config(resources)
-
-        task.set_resources({resources})
+        resources_config = config.pop('resources', None)
+        # Translate accelerators field to potential multiple resources.
+        if resources_config and resources_config.get(
+                'accelerators') is not None:
```

Comment:
If we don't have a way to take a set of resources not only having the accelerators different, e.g. `{Resources(sky.GCP()), Resources(sky.AWS())}`, our python API for spot instances will fail. As our `spot_launch` relies on the dumping and reloading for the yaml file. Should we support the list of resources support in yaml as well?
(We may not need to expose this to the user at the moment if we are not sure with the interface yet, but this should enable us to dump and reload any resources specification in python API)
```
resources:
  any_of:
    - cloud: AWS
       accelerator: A10g
    - cloud: GCP
       accelerator: L4
```
and for unordered list:
```
resources:
  ordered:
    - cloud: AWS
       accelerator: A10g
    - cloud: GCP
       accelerator: L4
```

Reply:
Done. Please see `examples/multi_resources.yaml`